components/card.css

```diff
@@ -0,0 +1,109 @@
+/* ECOM CARD */
+.vertical-card {
+	margin: 1rem;
+	display: flex;
+	position: relative;
+	flex-direction: column;
+	width: 25rem;
+	padding: 1rem;
+	-webkit-box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
```

Comment:
We don't need to add a vendor prefix unless you are trying to support a very old version browser.

Reply:
Okayy